tests/model_serving/model_server/maas_billing/conftest.py

```diff
@@ -458,3 +486,57 @@ def maas_models_response_for_actor(
         base_url=base_url,
         headers=maas_headers_for_actor,
     )
+
+
+@pytest.fixture(scope="class")
+def maas_inference_service_tinyllama(
+    admin_client: DynamicClient,
+    unprivileged_model_namespace: Namespace,
+    model_service_account: ServiceAccount,
+) -> Generator[LLMInferenceService, None, None]:
+    """
+    TinyLlama S3-backed LLMInferenceService wired through MaaS for tests.
+    """
+    with (
+        create_llmisvc(
+            client=admin_client,
+            name="llm-s3-tinyllama",
+            namespace=unprivileged_model_namespace.name,
+            storage_uri=ModelStorage.TINYLLAMA_S3,
+            container_image=ContainerImages.VLLM_CPU,
+            container_resources={
+                "limits": {"cpu": "2", "memory": "12Gi"},
+                "requests": {"cpu": "1", "memory": "8Gi"},
+            },
+            service_account=model_service_account.name,
+            wait=True,
+            timeout=Timeout.TIMEOUT_15MIN,
+        ) as llm_service,
+        patch_llmisvc_with_maas_router(llm_service=llm_service),
+    ):
+        llmd_resource = llm_service.instance.to_dict()
+        spec = llmd_resource.get("spec", {})
+        model_spec = spec.get("model", {})
+
+        storage_uri = model_spec.get("storageUri") or model_spec.get("storage_uri") or model_spec.get("uri")
+        assert storage_uri == ModelStorage.TINYLLAMA_S3, (
+            f"Unexpected storage_uri on TinyLlama LLMInferenceService: {storage_uri}"
+        )
+
+        status = llmd_resource.get("status", {})
```

Comment:
Please don't use dict here, you should save `llm_service.instance` and then use status property directly
```
llm_service_instance = llm_service.instance
llm_service_instance.status
```